controllers/userController.js

```diff
@@ -16,6 +17,51 @@ exports.getAllUsers = async (req, res) => {
   }
 }
 
+exports.getUsers = async (req, res) => {
+
+  //get the current user id from the token
+  const token = getEvenToken(req)
+  const decoded = decodeTkn(token, req.config.JWT_KEY)
+  const id = decoded[constants.idClaim]
+
+  //check if the current user is an admin
+  let isAdmin = req.user.admin
+  
+  if(!isAdmin) {
+    //get the data of the current user
+    try {
+      const userData = await req.db.User.findById(id)
```

Comment:
this should be `await req.db.User.findById(ObjectId(req.user._id))`

Reply:
the req.user object doesn't seem to have the _id field, but I think I should use req.user['user:id']